### DISCUSSION TITLE
Allow only existing Clients for DoH, is it possible?

### DISCUSSION THREAD

**Discussion by ghostersk** [AdguardTeam/AdGuardHome#6885] [Q&A]
I just found out that I can specify unique identifier for DNS over HTTPS, by adding some client name after dns-query
I was long time strugling to identify what device is making the query.

The question I have is if I can block all HTTPS queries if the client does not exist?
Maybe if it is `domain.com/dns-query` too
And allowed existing queries like `domain.com/dns-query/existing-client`

Edit:
I just noticed this in the DNS section, however I am not sure if this is what I want to use.
As I want to block all non existing clients on DNS over HTTPS, but i do not want to touch any dns provided by #53 port.
As I am using Traefik proxy in docker, it hiding the ip of real client behind the proxy or container ip if I am using normal dns from wireguard network.

I would like to apply this client filter only for DNS over HTTPS
![Screenshot_20240404_073601](https://github.com/AdguardTeam/AdGuardHome/assets/68815071/5808330d-843b-4840-9445-6ae416515f00)

**Comment by ainar-g** [comment]
There's only one set of Access settings for all protocols, so you cannot currently prohibit access for a set of ClientIDs only on one protocol.
